compiler.go

```diff
@@ -696,6 +731,8 @@ func (s *scope) finaliseVarAlloc(stackOffset int) (stashSize, stackSize int) {
 							i.idx = idx
 						case *resolveMixed:
 							i.idx = idx
+						case loadIndirect:
+							// do nothing
```

Comment:
It would be good to know _why_ do nothing is fine in this case.

Reply:
```suggestion
							// nothing needs to be changed
```